src/DependenciesPackager/Program.cs

```diff
@@ -184,7 +189,7 @@ static int Main(string[] args)
                 packagesVersion,
                 useCli,
                 quiet,
-                runtimes,
+                runtime,
```

Comment:
Why this change? We want to support generating packages for both bitnesses x86 and x64 in zip package. If there isn't anything compelling you to change this, please leave as it is.